### DISCUSSION TITLE
How to pass api parameters to call this extension?

### DISCUSSION THREAD

**Discussion by keymonzhang** [NVIDIA/Stable-Diffusion-WebUI-TensorRT#212] [Q&A]
I hope to use webui using the api interface calling method. In Text2Img or Img2Img, how should I pass parameters to use this extension?

**Comment by Pista007** [comment]
Any answer for this?

**Comment by DNArtbill** [comment]
yeah would love to have this option as well great to have this speed a real shame theres no api support are there any sd servers that allow us to use TensorRT true an api call ? we can generate the models in auto1111
